### ISSUE TITLE
Summoners

### ISSUE THREAD

**Issue by ahnafabid66** [Gelbpunkt/IdleRPG#190]
Lets summoners and ritualists actually summon something
like maybe a demon or skeletons
just something to add a lil more fun into the class

**Comment by SMar7-Fox** [comment]
And what should they even do?

**Comment by Gelbpunkt** [comment]
This is Ritualist, not Paragon/Warrior/Mage

**Comment by Slavis-from-Idle** [comment]
I think that developing that summoning theme of ritualist is a great idea. Lets think - summons are usualy made for knowledge and power. That means, it could work in similar fashion to pet mechanics or missions (few tiers of rituals, the more complicated it is, the more time need to be spend on preparations but also, the reward is greater). The rewared could be experience (summoned entity decides to share knowledge), item / loot (bargain with the summon for some rare magic artifacts / items made on other planes of existence) or perhaps some random, temporary (2-3h?) stat boost (attack / defence / luck / etc? Result of bargain for power with entity - perhaps sth must be offered in return for greater bonuses?)
Or - however, that is more complicated topic - grant some kind of 'wish' that will allow to temporary influence the world - like protection from thieves, giving some minor buffs / debuffs to others, to guild etc.?

The level of ritual can be dicteted by arcane / plane / summoning knowledge of the caster that will be evolved by similar fashion to the pet. Like, Novice at first, Conjurer later, Planewalker, Master Summoner etc.

However, it should have sth, that will distunguish it from pets, so it will not be just a copy paste. Perhaps Summoners should have option to make a PvP battle between their summons? Or maybe a joined ritual can be performed, like a Summoning Circle, it would work just like guild adventure or raid?

To sum up, above I have presented some fast ideas, that cross my mind minutes ago. Topic definitely should be developed, since it has got a lot of potential for fun factor of this class.

**Comment by Gelbpunkt** [comment]
The class does enough already in my view. Should probably rename the evolutions to something that fits better

**Comment by maxx0911** [comment]
At least give them a fun command that summons skeletons. They could randomly choose a dance gif like [this one](https://media3.giphy.com/media/R7mR1YmHaOwlW/giphy.gif?cid=790b7611e4b3c3fe19a4c8626552278f0ea680e3ca62f5ca&rid=giphy.gif).

**Comment by Slavis-from-Idle** [comment]
Perhaps that can be material for a next class? Because the idea of a summoner seems to be worth it.
